src/components/generic/movieModal/movieModal.tsx

```diff
@@ -1,32 +1,10 @@
 import * as React from 'react'
 import {FC} from 'react'
 
-import ModalContent from './ModalContent'
+import {useModal} from '@/context/modal-context'
+import {useMovie} from '@/api/movie_api'
+import {ModalContent} from './ModalContent'
 
-interface Data {
-  actors: string
-  description: string
-  directors: string
-  genres: string[]
-  id: string
-  imdb: string
-  poster_big: string
-  rating: number
-  title: string
-  trailer: string
-  year: number
-  items: []
+export const MovieModal: FC = () => {
```

Comment:
Do you even need this component if it just renders another component without doing anything?

Reply:
After refactor. No.

I got it removed.